core/components/ui/header/search.tsx

```diff
@@ -116,7 +121,18 @@ const Search = ({ initialTerm = '', logo, onSearch }: Props) => {
             <div className="grid h-[92px] w-full grid-cols-5 items-center">
               <div className="me-2 hidden lg:block lg:justify-self-start">
                 <Link className="overflow-hidden text-ellipsis py-3" href="/">
-                  {logo}
+                  {typeof logo === 'object' ? (
+                    <BcImage
+                      alt={logo.altText}
+                      className="max-h-16 object-contain"
+                      height={32}
+                      priority
+                      src={logo.src}
+                      width={155}
+                    />
+                  ) : (
+                    <span className="truncate text-2xl font-black">{logo}</span>
+                  )}
```

Comment:
🍹 I understand why we have this now, but the logo will be removed from the quick search component eventually with Soul. I think it's fine to keep it for this PR, but just calling it out because it'll change the props API.

Reply:
Yeah, APIs will be simplified 🙏 .